lib/lokka/models/entry.rb

```diff
@@ -121,25 +121,27 @@ def description
     desc.gsub(%r{<[^/]+/>}, ' ').gsub(%r{</[^/]+>}, ' ').gsub(/<[^>]+>/, '').html_safe
   end
 
-  class << self
+  module FinderstWithScope
     def _default_scope
       { order: :created_at.desc }
     end
 
-    def first_with_scope(limit = 1, query = DataMapper::Undefined)
+    def first(limit = 1, query = DataMapper::Undefined)
```

Comment:
Don't need to name different one for finders as we use `Module.prepend` so that original method's behavior can changed by context.